pkg/pkg.ml

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env ocaml
+
```

Comment:
we can probably get rid of this file, since we'd need to use dune-release instead of topkg for publishing new releases

Reply:
I'm going to leave this since I don't know what it does. We can remove it in another PR if it's unnecessary.